src/test/java/JunitPracticalTask2.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.logging.Logger;
+
+public class JunitPracticalTask2 {
+    private static Logger LOGGER = Logger.getLogger(JunitPracticalTask2.class.getName());
```

Comment:
Would be useful for you to see how all of these methods work with `@ParameterizedTest`